docs/jade-getting-started.md

```diff
@@ -217,14 +222,39 @@ git clone https://github.com/broadinstitute/datarepo-helm
 git clone https://github.com/broadinstitute/datarepo-helm-definitions
 ```
 
-Create your datarepo helm definition:
-1. In `datarepo-helm-definitions/dev` directory, copy an existing developer
+## 8. [DRAFT] Set up your Development Environment
+The goal of this step is set up some of the basic components of your development
+environment.  You'll actually spin up this instance on broad-jade-dev in next step.
+
+Note: These instructions have not been tested yet! This may be a good step to
+pair on with another jade team member. There is a video of us walking through
+these steps in our [Jade Googe Drive Folder](https://drive.google.com/drive/folders/1JM-_M0qsX6eXocyPc9TB7ivCKJTji3dX?usp=sharing). 
```

Comment:
```suggestion
> These instructions have not been tested yet! This may be a good step to
pair on with another Jade team member. There is a video of us walking through
these steps in our [Jade Google Drive Folder](https://drive.google.com/drive/folders/1JM-_M0qsX6eXocyPc9TB7ivCKJTji3dX?usp=sharing). 
```